From 83f021b7fce316cf47a6727ba7c1f85233d889e2 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 27 Oct 2014 17:15:40 +0800
Subject: [PATCH] anaconda: use tty7 to instead tty6 as console

While systemd enabled, if Xorg choose tty6 as console, the console
access will fail while receiving a SIGHUP at Xorg first init time.

Use tty7 could avoid the issue, the reason is unknown.

Upstream-Status: Inappropriate [WR installer specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 anaconda | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/anaconda b/anaconda
index 53e421e..d2330ea 100755
--- a/anaconda
+++ b/anaconda
@@ -501,7 +501,7 @@ def setupDisplay(anaconda, opts):
             try:
                 proc = subprocess.Popen(["Xorg", "-br",
                                          "-logfile", "/tmp/X.log",
-                                         ":1", "vt6", "-s", "1440", "-ac",
+                                         ":1", "vt7", "-s", "1440", "-ac",
                                          "-nolisten", "tcp", "-dpi", "96",
                                          "-noreset"],
                                          close_fds=True,
-- 
1.9.1

